config/notifiers.go

```diff
@@ -825,3 +834,42 @@ func (c *MSTeamsConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
 
 	return nil
 }
+
+type JiraConfig struct {
+	NotifierConfig `yaml:",inline" json:",inline"`
+	HTTPConfig     *commoncfg.HTTPClientConfig `yaml:"http_config,omitempty" json:"http_config,omitempty"`
+
+	APIURL *URL `yaml:"api_url,omitempty" json:"api_url,omitempty"`
+
+	Project      string   `yaml:"project,omitempty" json:"project,omitempty"`
+	Summary      string   `yaml:"summary,omitempty" json:"summary,omitempty"`
+	Description  string   `yaml:"description,omitempty" json:"description,omitempty"`
+	StaticLabels []string `yaml:"static_labels,omitempty" json:"static_labels,omitempty"`
+	GroupLabels  []string `yaml:"group_labels,omitempty" json:"group_labels,omitempty"`
+	Priority     string   `yaml:"priority,omitempty" json:"priority,omitempty"`
+	IssueType    string   `yaml:"issue_type,omitempty" json:"issue_type,omitempty"`
+
+	ReopenTransition  string   `yaml:"reopen_transition,omitempty" json:"reopen_transition,omitempty"`
+	ResolveTransition string   `yaml:"resolve_transition,omitempty" json:"resolve_transition,omitempty"`
+	WontFixResolution string   `yaml:"wont_fix_resolution,omitempty" json:"wont_fix_resolution,omitempty"`
+	ReopenDuration    Duration `yaml:"reopen_duration,omitempty" json:"reopen_duration,omitempty"`
```

Comment:
why not model.Duration?

Reply:
I used Duration for constancy reasons. It's used inside the PushoverConfig. But I guess PushoverConfig can model.Duration as well.